platform/android/src/native_map_view.cpp

```diff
@@ -196,6 +201,14 @@ void NativeMapView::render() {
     }
 
     deactivate();
+
+    if (currentContext != context && currentContext != EGL_NO_CONTEXT) {
```

Comment:
:+1: 

But I think it is better `deactivate()` if there is no prior GLContext. Gonna change that.